### PR TITLE
Add explicit account server URL to `nsc push` step

### PR DESCRIPTION
**Pull request by bruth**

### CODE REVIEW

**Comment by gcolliso** [comment]
@aricart or @matthiashanel would you be able to review, please?

File: nats-tools/nas/dir_store.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Success! - added account "B"
 With the account and a couple of users in place, let's push all the accounts to the nats-account-server:
 
 ```text
-> nsc push -A
+> nsc push -u http://localhost:9090/jwt/v1/ -A
```
Comment (aricart) [inline nats-tools/nas/dir_store.md:66]: This is only required if the operator doesn't set an account server URL. 
To simplify the workflow, you can edit the operator:
```
❯ nsc edit operator -u http://localhost:9090/jwt/v1
[ OK ] set account jwt server url to "http://localhost:9090/jwt/v1"
[ OK ] edited operator "AAA"

```

**Review by aricart — CHANGES_REQUESTED** [review]
LGTM but the reason why the url is required is because the operator is not correctly configured. An account server URL should be set. We should add the bit of the comment, and then show that even if not set, specifying the URL will work.

File: nats-tools/nas/dir_store.md (hunk 1 above)
Comment (matthiashanel) [reply nats-tools/nas/dir_store.md:66]: I concur with @aricart

Comment (bruth) [reply nats-tools/nas/dir_store.md:66]: Thanks, understood. It was not clear when reading through the docs that this was necessary or preferred over the `push -u` option. That is why I wanted to raise it here. Of course feel free to update the docs however is appropriate.

**Comment by bruth** [comment]
@aricart Updated the change to include the note.

**Review by matthiashanel — APPROVED** [review]
LGTM

**Review by aricart — APPROVED** [review]
LGTM

**Comment by bruth** [comment]
Thanks all, changes look good to me as well.

**Review by gcolliso — COMMENTED** [review]
lgtm - thanks all!

**Pull request merged**